visualization/app/codeCharta/ui/mapTreeView/mapTreeView.component.ts

```diff
@@ -8,6 +8,7 @@ import {
 } from "../../state/store/appSettings/sortingOrderAscending/sortingOrderAscending.service"
 import { SortingOptionService, SortingOptionSubscriber } from "../../state/store/dynamicSettings/sortingOption/sortingOption.service"
 import { NodeMetricDataService } from "../../state/store/metricData/nodeMetricData/nodeMetricData.service"
+import {klona} from "klona";
```

Comment:
Nit: please use prettier to format the code before commiting. We might want to add that step to the commit hook. It's a script in the main package.json (main folder) and not in the visualization. I am also fine to copy the script as long as we trigger it ;-)

Reply:
I have added a regression (unit) test. 
I did not know how to test it with an end to end test because the canvas DOM element does not change.

Reply:
> Nit: please use prettier to format the code before commiting. We might want to add that step to the commit hook. It's a script in the main package.json (main folder) and not in the visualization. I am also fine to copy the script as long as we trigger it ;-)

I created a new issue for that: #1420

Reply:
@BridgeAR the husky hook is located in root and will trigger for visualization as well (since visualization is a child of the project). Either commit through the terminal or if you use the GUI in IntelliJ check the `run commit hooks` option.
And don't forget to `npm i` on root.
<img width="356" alt="Bildschirmfoto 2020-10-22 um 12 45 13" src="https://user-images.githubusercontent.com/12533626/96861477-72308680-1464-11eb-8cdf-9f115badab47.png">